API/README.md

```diff
@@ -0,0 +1,18 @@
+## Launch backend server locally
+
+Open command line and cd to `API` folder create virtual env
+
+```$ python3 -m venv venv```
+
+Activate environment
+```source venv/bin/activate```
```

Comment:
Can we additionally add On Windows 10 command prompt it is: 
```venv/Scripts/activate```

Reply:
Sure, good catch!